source/shared/cpp/ObjectModel/CaptionSource.h

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#pragma once
+
+#include "pch.h"
+#include "BaseCardElement.h"
+#include "ElementParserRegistration.h"
+#include "Util.h"
+
+namespace AdaptiveCards
+{
+class CaptionSource
```

Comment:
How is this class any different from `MediaSource` ?:) It's identical. In order to not copy the same code, may we can extract the base class and have two children `MediaSource` and `CaptionSource` to differentiate?:)

Reply:
Makes sense, done!